__tests__/app.test.js

```diff
@@ -35,3 +35,17 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET 200: receive a description of all other endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(Object.keys(response.body.api)).toEqual([
```

Comment:
This will break when you add your next endpoint, for this you can just require your endpoints json file in your test and just test that it is being sent when you make a request.